src/RootedTrees.jl

```diff
@@ -1413,6 +1415,42 @@ function butcher_representation(t::RootedTree, normalize::Bool = true)
     return result
 end
 
+"""
+    elementary_differential(t::RootedTree)
+
+Returns the elementary differential as a LaTeXString.
+
+"""
+function elementary_differential(t::RootedTree)
+    return latexstring(rec_elementary_differential(t))
+end
+
+"""
+Function used to go recursively through the RootedTree to generate the elementary differential of the tree.
+"""
+function rec_elementary_differential(t::RootedTree)
+    subtree_strings = []
```

Comment:
This generates a `Vector{Any}`. Thus, anything accessing it will have some overhead since it is not type-stable. Do you know the type of stuff stored in `subtree_strings`, e.g., `String`? If so, it would be better to set this right now, e.g., `subtree_strings = String[]`.